src/controller/organization.controller.ts

```diff
@@ -17,6 +19,60 @@ import { OrganizationsGetRequestBody } from "@/models/organization/Organizations
 import { OrganizationsEmailRequestBody } from "@/models/organization/OrganizationsEmailRequestBody";
 import { sendSecretMail } from "@/components/SendEmail";
 import { Prisma } from "@prisma/client";
+import { OrganizationsEmailConfirmBody } from "@/models/organization/OrganizationsEmailConfirmBody";
+import { verifyEmailToken } from "@/service/manageVerifyToken";
+import { TokenExpiredError } from "jsonwebtoken";
+
+interface JwtPayload {
+  userId: string;
+  organizationId: string;
+}
+export const confirmOrganizationEmail = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+) => {
+  const { AccessToken } = req.body;
+  try {
+    const { userId, organizationId } = verifyEmailToken(
+      AccessToken
+    ) as JwtPayload;
+    const organizationsEmailConfirmBody = new OrganizationsEmailConfirmBody(
+      userId,
+      organizationId
+    );
+    await updateEmailVerifyStatus(
+      organizationsEmailConfirmBody.userId,
+      organizationsEmailConfirmBody.organizationId
+    );
+    res.status(200).send(
+      new ResponseBody({
+        message: ORGANIZATIONS_EMAIL_CONFIRM_SUCCESS,
+      })
+    );
+  } catch (e) {
+    if (typeof e === "string") {
+      res.status(400).send(new ResponseBody({ message: e }));
+      return;
+    } else if (e instanceof Prisma.PrismaClientKnownRequestError) {
+      res.status(409).send(
+        new ResponseBody({
+          message: e.message,
+        })
+      );
+      return;
+    } else if (e instanceof TokenExpiredError) {
+      res.status(401).send(
+        new ResponseBody({
+          message:
+            "이메일 인증 기간이 만료되었습니다. 다시 인증 요청을 보내주세요.",
+        })
+      );
+    }
```

Comment:
early return을 하는 경우 보통 `else if` 대신 `if`만 사용합니다. 그리고 마지막 조건문에 return이 빠진것같네요
```suggestion
    if (typeof e === "string") {
      res.status(400).send(new ResponseBody({ message: e }));
      return;
    } 
    if (e instanceof Prisma.PrismaClientKnownRequestError) {
      res.status(409).send(
        new ResponseBody({
          message: e.message,
        })
      );
      return;
    } 
    if (e instanceof TokenExpiredError) {
      res.status(401).send(
        new ResponseBody({
          message:
            "이메일 인증 기간이 만료되었습니다. 다시 인증 요청을 보내주세요.",
        })
      );
      return;
    }
```